SilKit/source/core/vasio/RingBuffer.hpp

```diff
@@ -0,0 +1,352 @@
+/* Copyright (c) 2022 Vector Informatik GmbH
```

Comment:
You can replace that with the SPDX Header, see the header in https://github.com/vectorgrp/sil-kit/blob/main/SilKit/source/core/vasio/ConnectPeer.cpp